components/_patterns/02-molecules/menus/_menu.twig

```diff
@@ -34,9 +32,7 @@
 
     {# Set classes #}
     {% set menu_class = menu_class|default('menu') %}
-    {% if not menu_modifiers %}
```

Comment:
Same as above. Here just in case we pass menu_modifiers to the template so that they aren't overwritten by the perhaps necessary array initialization.